src/main/java/network/brightspots/rcv/TabulatorSession.java

```diff
@@ -336,8 +336,11 @@ private boolean setUpLogging(ContestConfig config) {
         encounteredSourceProblem = true;
       } catch (CvrParseException e) {
         encounteredSourceProblem = true;
+      } catch (Exception exception) {
+        Logger.severe("Unexpected error parsing source file %s", cvrPath);
```

Comment:
If you're going to catch a general exception like this (which is typically not recommended), I'd at least make sure you include the details in the error message, e.g.:

```
} catch (Exception e) {
  Logger.severe("Unexpected error parsing source file: %s\n%s", cvrPath, e.toString());
```